### ISSUE TITLE
Here is a link to updated zipfiles

### ISSUE THREAD

**Issue by dkfurrow** [kelyvin/Google-Messages-For-Desktop#47]
Old build warnings showed up on my screen, so rebuilt file in forked version in my user area.
Please find files [here](https://github.com/dkfurrow/Google-Messages-For-Desktop)

**Comment by PirateIncognito** [comment]
Thank you!!

**Comment by jcritchjr** [comment]
> Old build warnings showed up on my screen, so rebuilt file in forked version in my user area. Please find files [here](https://github.com/dkfurrow/Google-Messages-For-Desktop)

The Windows Tray version does NOT Extract. "Unexpected error extracting".

[UnexpectedErrorExtracting.pdf](https://github.com/kelyvin/Google-Messages-For-Desktop/files/7919509/UnexpectedErrorExtracting.pdf)

**Comment by dkfurrow** [comment]
Hi @jcritchjr;
I'm afraid I can't replicate the problem with unzipping.  My steps: (1) Download [file](https://github.com/dkfurrow/Google-Messages-For-Desktop/raw/master/Windows/google-messages-windows-tray_v1.3.0.zip) (2) Unzip using [7zip](https://www.7-zip.org/download.html).  So I did get an 'old build detected' from the files that were out there, so I rebuilt and pushed new zipfiles in my repository, and the updated files are running correctly, at least for me.
